### ISSUE TITLE
How long does it take to train ?

### ISSUE THREAD

**Issue by cvbird** [zzr-idam/UVM-Net#4]
Hi zzr, thank you for sharing this repo. I wonder how long does it take to train your model on the RESIDE-OUT dataset. I trained with 4 GPUs, and it took around 9 hours for one epoch on RESIDE-OUT dataset. I wonder whether the time cost matches your experiment, as you announced the training setting was consisted with Vision Transformers for Single Image Dehazing which trained 30 epochs. So in my experiment, it may take 30*9=270 hours = 11.25 days (too looooooooooong).

Thanks again.

**Comment by zzr-idam** [comment]
10 epoch is used&nbsp;

----------

该邮件从移动设备发送


--------------原始邮件--------------
发件人："cvbird ***@***.***&gt;;
发送时间：2024年3月1日(星期五) 晚上11:11
收件人："zzr-idam/UVM-Net" ***@***.***&gt;;
抄送："Subscribed ***@***.***&gt;;
主题：[zzr-idam/UVM-Net] How long does it take to train ? (Issue #4)
-----------------------------------

 

 
Hi zzr, thank you for sharing this repo. I wonder how long does it take to train your model on the RESIDE-OUT dataset. I trained with 4 GPUs, and it took around 9 hours for one epoch on RESIDE-OUT dataset. I wonder whether the time cost matches your experiment, as you announced the training setting was consisted with Vision Transformers for Single Image Dehazing which trained 30 epochs. So in my experiment, it may take 30*9=270 hours = 11.25 days (too looooooooooong).
 
Thanks again.

**Comment by zzr-idam** [comment]
We were using 8 V100s and it took just 3 days. If you think the loss is not dropping anymore, try to stop and test it.

**Comment by zzr-idam** [comment]
You can use our model to test it on a small number of datasets such as set5 to evaluate its capabilities.

**Comment by zzr-idam** [comment]
Currently, our model is used in the CVPR2024 workshop.

**Comment by zzr-idam** [comment]
> Hi zzr, thank you for sharing this repo. I wonder how long does it take to train your model on the RESIDE-OUT dataset. I trained with 4 GPUs, and it took around 9 hours for one epoch on RESIDE-OUT dataset. I wonder whether the time cost matches your experiment, as you announced the training setting was consisted with Vision Transformers for Single Image Dehazing which trained 30 epochs. So in my experiment, it may take 30*9=270 hours = 11.25 days (too looooooooooong).
> 
> Thanks again.

What is your GPU version?

**Comment by cvbird** [comment]
> > Hi zzr, thank you for sharing this repo. I wonder how long does it take to train your model on the RESIDE-OUT dataset. I trained with 4 GPUs, and it took around 9 hours for one epoch on RESIDE-OUT dataset. I wonder whether the time cost matches your experiment, as you announced the training setting was consisted with Vision Transformers for Single Image Dehazing which trained 30 epochs. So in my experiment, it may take 30*9=270 hours = 11.25 days (too looooooooooong).
> > Thanks again.
> 
> What is your GPU version?

V100 32G too. By the way, did you change the batch size ?
